app/templates/index.html

```diff
@@ -32,7 +32,14 @@
 
     <div class="profile">
         <div id="profile-picture" class="profile-picture">
-            <img src="./static/img/logo.jpg">
+            <!-- Will's Profile Picture -->
```

Comment:
Do we need to keep Will's? I assumed that was the one we would be replacing.

Reply:
In fact I'm not sure, what do you think? should we keep it or remove it? 👀

Reply:
I say we get rid of it to make it a little more our own 🪓

Reply:
Yes, let's remove it. Your portfolio is yours only 😉

Reply:
I'm going to remove Will's photo and add an avatar logo for now while you all upload your photos as well 🙌🏻